### ISSUE TITLE
Same nuget package works in VS Enterprise but doesnt in VS Code via CakeScript.

### ISSUE THREAD

**Issue by JayBeAl** [cake-build/cake#2226]
dotnet.exe, nuget.exe (Visual Studio Code - Cake Script)
and
Visual Studio Enterprise 2017

NuGet version: 4.7.0.5148
dotnet.exe: 2.1.302
VS version : 15.7.5
Cake version: 0.29.0

**Detailed repro steps:**
SHFB(Sandcastle Help File Builder): A tool to create documentation out of XML-Comments

_Installed following nuget packages with the Visual Studio nuget-package-manager:_
EWSoftware.SHFB --> version 2018.7.8
EWSoftware.SHFB.NetFramework --> version 4.7.2
-> worked properly and i had no problems to use it.

_Now i used a cake script in Visual Studio Code to automate documentation creation._
i tried to add both packages like this:
#tool nuget:?package=EWSoftware.SHFB&version=2018.7.8
#tool nuget:?package=EWSoftware.SHFB.NETFramework&version=4.7.2

on command "./build" the first package installed without problems.
the second package didn't, i got this error: "E
rror: Failed to install tool 'EWSoftware.SHFB.NETFramework'"

I tried all possible Versions of the .NETFramework nuget package (4.6.0; 4.6.1; 4.6.2; 4.7.0; 4.7.2)
Same Error on every Version.

I also contacted the product owner but he couldn't help me.
And i already posted this issue in the nuget issue forum.

So my Question is, why is the same package working for VS and not working for cakescript in VS Code?

Here my cakescript etc. to reproduce my problem in VS Code:
[cakescript.zip](https://github.com/cake-build/cake/files/2252657/cakescript.zip)

TL;DR:
Installing Nugets:
EWSoftware.SHFB --> version 2018.7.8
EWSoftware.SHFB.NetFramework --> version 4.7.2
Works in VS Enterprise.
Doesnt work via Cakescript in VS Code.

Thanks in advance!

Greetings
Jay

**Comment by JayBeAl** [comment]
@gep13 thanks for the fast answer. As far as i know the second package is to specifiy the platform on which the first one is running.

The Description for the nuget EWSoftware.SHFB.NetFramework is:
This package contains the .NETFramework/.NETCore/.NETStandard reflection information data set for use with the EWSoftware.SHFB package.

I now deleted the second package in visual studio and it seems to still work. Im a bit confused now, cause in the tutorial it was said to install both packages. 

Well but that explains why there is no exe in the tool folder if installed and why Cake is throwing an Error.

Thank you

**Comment by gep13** [comment]
If you find that you "need" that package, then I would suggest using the NuGetInstall alias to install the required NuGet package:

https://cakebuild.net/api/Cake.Common.Tools.NuGet/NuGetAliases/343A93DB

For now, I am going to go ahead and close this issue.
